platform/ios/docs/guides/For Style Authors.md

```diff
@@ -253,9 +271,9 @@ In style JSON | In Objective-C | In Swift
 
 Each property representing a layout or paint attribute is set to an
 `MGLStyleValue` object, which is either an `MGLConstantStyleValue` object (for
-constant values) or an `MGLStyleFunction` object (for zoom level functions). The
-style value object is a container for the raw value or function parameters that
-you want the attribute to be set to.
+constant values) or an `MGLStyleFunction` subclass. The style value object is a
+container for the raw value or function parameters that you want the attribute
+to be set to.
```

Comment:
Setting style functions is a subtask of configuring the map content’s appearance. I think the table mapping function types to MGLStyleFunction subclasses and MGLInterpolationMode values should be moved down here, to help progress the reader from simpler concepts to more advanced concepts.